templates/security/register_user.html

```diff
@@ -20,4 +20,4 @@ <h1>Register</h1>
         </div>
     </div>
 </div>
-{% endblock body %}
```

Comment:
line end changed

Reply:
this was already cahnged accoding to my repo.

Reply:
well it is changed according to this pull request and it is an irreverent change. Can you remove this please as it makes it harder to review the actual logical changes to the app.